.github/pull_request_template.md

```diff
@@ -24,7 +24,6 @@ Please describe the tests that you ran to verify your changes. Provide instructi
 
 # Checklist:
 
-- [ ] My code follows the [javascript standard style](https://standardjs.com/)
```

Comment:
technically we are pulling in rules from standardjs (plus a few extras) so it would be useful to keep it in here if anyone needs a reference. maybe it could be changed to `my code does not produce any linting errors`?

Reply:
Cool, will add it back in in a future PR